### DISCUSSION TITLE
How to create overlap or stacking in the layers for text annotation?

### DISCUSSION THREAD

**Discussion by Roxanne0330** [inception-project/inception#4099] [Q&A]
I have a problem with creating overlapping or stacking in different layers for text annotation. Look forward to your reply soon.
<img width="1430" alt="Bildschirmfoto 2023-07-14 um 12 55 42" src="https://github.com/inception-project/inception/assets/139484750/4cdc4b60-e911-42a8-af1f-4fc254b78495">
<img width="1121" alt="Bildschirmfoto 2023-07-14 um 12 56 09" src="https://github.com/inception-project/inception/assets/139484750/56a4a1e9-bd85-4c71-b0e3-de4f849c6247">

**Comment by reckart** [comment]
Change the behavior **Overlap** in the middle column from **None** to **Any** and use the save button at the top of the middle column.
